index.html

```diff
@@ -0,0 +1,91 @@
+                </a></div>
+            </div>
+        </div>
+        
+    </section>
+    <section class="sub-section" id="about-me">
+        <div class="information">
+            <h2>About Me</h2>
+            <p>
+                Hello! My name is Reid and I am a software developer.
+            </p>
+        </div>
+        <div class="headshot-container">
+
+            <img class="headshot" src="src/images/technologist-emoji.png" alt="blonde technologist emoji"/>
+        </div>
+
+    </section>
+    <section class="sub-section-alternative" id="projects">
+        <h2>Projects</h2>
+        <div class="project-container">
+            <div class="project-card">
+                <img class="project-image" src="src/images/seedling.png" alt=""/>
+                <h3>Adagrams</h3>
+                <p class="subtext">One of the first projects I completed.</p> 
+                <hr/>
+                <p class="subtext"><a class="project-link" href="https://github.com/reidhowdy/adagrams-py">View here</a></p>
+            </div>
+            <div class="project-card">
+                <img class="project-image" src="src/images/seedling.png" alt=""/>
+                <h3>Viewing Party</h3>
+                <p class="subtext">Practice accessing nested data structures.</p> 
+                <hr/>
+                <p class="subtext"><a class="project-link" href="https://github.com/reidhowdy/viewing-party">View here</a></p>
+            </div>
+            <div class="project-card">
+                <img class="project-image" src="src/images/seedling.png" alt=""/>
+                <h3>Swap Meet</h3>
+                <p class="subtext">Practice using classes in Python.</p> 
+                <hr/>
+                <p class="subtext"><a class="project-link" href="https://github.com/reidhowdy/swap-meet">View here</a></p>
+            </div>
+            <div class="project-card">
+                <img class="project-image" src="src/images/seedling.png" alt=""/>
+                <h3>Retro Video Store</h3>
+                <p class="subtext">Practice building an API in Python</p> 
+                <hr/>
+                <p class="subtext"><a class="project-link" href="https://github.com/reidhowdy/retro-video-store">View here</a></p>
+            </div>
+        </div>
+    </section>
+    <footer id="contact-me"></footer>
+    
+
+    <script srd="src/app.js"></script>
```

Comment:
Notice the typo in the attribute (srd instead of src). But we didn't want to use scripts in this project anyway!